src/app/main/sidebar/sidebar.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-sidebar',
+  templateUrl: './sidebar.component.html',
+  styleUrls: ['./sidebar.component.css']
+})
+export class SidebarComponent {
+
+  openLink(pdfUrl: string) {
+    window.open(pdfUrl, '_blank');
```

Comment:
Nie jestem pewien, czy to jest na pewno bezpieczne otwieranie tak linku z _blank w `window.open`, możesz sprawdzić.